packages/rule-https-only/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "description": "sonarwhal rule that checks if your site is using https and if it has Mixed Content.",
+  "devDependencies": {
+    "@types/debug": "0.0.30",
+    "@types/node": "8.0.14",
```

Comment:
Do you need this? We removed it from everywhere else as it creates a lot of problems.

Reply:
We have it in the root one, maybe it is not necessary here.